.agents/skills/github/SKILL.md

```
@@ -0,0 +1,166 @@
+
+## 🛑 MANDATORY PRE-CHECK & SAFETY DIRECTIVES
+
+1. **Environment Check:** Always execute this verification script BEFORE performing any GitHub operations:
+
+```bash
+   bash ./scripts/ai-gh-check.sh
+```
+
+_If it fails, output the instructions provided by the script and STOP._
+
+2. **Mandatory Attribution Signature:**
```

markdownlint-cli2 (0.23.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)